ext/hash/tests/hash_copy_001.phpt

```diff
@@ -301,7 +301,7 @@ string(16) "bebc746a33b6ab62"
 string(16) "893899e4415a920f"
 string(5) "joaat"
 string(8) "aaebf370"
-string(8) "513479b4"
+string(8) "836fb0e5"
```

Comment:
Why has the expect changed?

Reply:
Bug fix for joaat.

Reply:
The joaat fixes were merged in another PR, so I'll just resolve these.